### PR TITLE
Finish the mobile version

### PR DESCRIPTION
**Pull request by Nezerwa**
>in this project I

- added input for name
- added input for email
- added textarea

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -224,5 +224,19 @@ <h3>Skills</h3>
...
+              <p class="footer-p">
+                If you have an application you are interested in developing,
+                 a feature that you need built or a project that needs coding.
+                  I’d love to help with it 
+              </p>
+              <form  class="feedback" action="https://formspree.io/html/" method="post">
+               <input class="name" type="text" placeholder="Enter your name" name="name" maxlength="30" required>
+               <input class ="email" type="email" placeholder="Enter your email" name="email" required>
+               <textarea rows="10" maxlength="500" placeholder="Write your message here"></textarea>
+               <input  class="get-in-touch" type="submit" value="Get In Touch" required>
```
Comment (ichala) [inline index.html]: - [ ] good job adding HTML validations 👏  , however as part of the requirements, the textarea should have a required attribute as well, kindly add that 🙏

File: index.html [hunk 2]
```diff
@@ -224,5 +224,19 @@ <h3>Skills</h3>
...
-</html>
\ No newline at end of file
+            <footer>
+              <h2 class="contact-me">Contact me</h2>
+              <p class="footer-p">
+                If you have an application you are interested in developing,
+                 a feature that you need built or a project that needs coding.
+                  I’d love to help with it 
+              </p>
+              <form  class="feedback" action="https://formspree.io/html/" method="post">
```
Comment (ichala) [inline index.html]: - [ ] Please note that the action link https://formspree.io/html/ is not a valid formspree URL, you have to create an account at [formspree ](https://formspree.io/register)and then generate your own id and then you need to add it to your form action, the correct link format should be like this: _**`https://formspree.io/f/{form_id}`**_ , where the {form_id} is the key generated in your formspree account , kindly fix that 🙏 

image

File: styles.css [hunk 3]
```diff
@@ -340,4 +340,83 @@ main {
...
+  color: #ebebff;
+  margin-left: 10%;
+}
+
+.feedback {
+  display: grid;
+  grid-template-columns: auto;
+  row-gap: 20px;
+  width: 447px;
+}
```
Comment (ichala) [inline styles.css:382]: - [ ] Kindly fix the overflow of your form, I would highly recommend removing the fixed width from `.feedback` and the error should be fixed 🙏 

HINT12

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @Nezerwa 
Good job so far! 
👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Html Validations works✔️ 
- No linters errors✔️

 ## Required Changes ♻️

- [ ] 🛑 **_Check the comments below_**

## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -224,5 +224,19 @@ <h3>Skills</h3>
...
+               <input class="name" type="text" placeholder="Enter your name" name="name" maxlength="30" required>
+               <input class ="email" type="email" placeholder="Enter your email" name="email" required>
+               <textarea rows="10" maxlength="500" placeholder="Write your message here"></textarea>
```
Comment (omarsalem7) [inline index.html]: - [ ] As stated by the previous reviewer kindly add the required attribute for the Textarea field.

File: index.html (hunk 2 above)
Comment (omarsalem7) [inline index.html]: - [ ] As stated before kindly register in [formspree](https://formspree.io/html/) and register and use the id that you get and replace the action link with the following one and do not forget to replace the form_id also `https://formspree.io/f/{form_id}`

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @Nezerwa , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

Almost near

 ## Highlights
✔️ All linter checks pass
✔️ Descriptive pull request

 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Nezerwa 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - No linter issues. ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**